qa/standalone/misc/ok-to-stop.sh

```diff
@@ -237,5 +237,53 @@ function TEST_0_mds() {
     kill_daemons $dir KILL mds.a
 }
 
+function TEST_0_osd() {
+    local dir=$1
+
+    CEPH_ARGS="$ORIG_CEPH_ARGS --mon-host=$CEPH_MON_A "
+
+    run_mon $dir a --public-addr=$CEPH_MON_A || return 1
+    run_mgr $dir x || return 1
+    run_osd $dir 0 || return 1
+    run_osd $dir 1 || return 1
+    run_osd $dir 2 || return 1
+    run_osd $dir 3 || return 1
+
+    ceph osd erasure-code-profile set ec-profile m=2 k=2 crush-failure-domain=osd || return 1
+    ceph osd pool create ec erasure ec-profile || return 1
```

Comment:
This doesn't work in Nautilus:

```
2020-01-29T19:46:40.536 INFO:tasks.workunit.client.0.smithi105.stderr:/home/ubuntu/cephtest/clone.client.0/qa/standalone/misc/ok-to-stop.sh:252: TEST_0_osd:  ceph osd erasure-code-profile set ec-profile m=2 k=2 crush-failure-domain=osd
2020-01-29T19:46:41.439 INFO:tasks.workunit.client.0.smithi105.stderr:/home/ubuntu/cephtest/clone.client.0/qa/standalone/misc/ok-to-stop.sh:253: TEST_0_osd:  ceph osd pool create ec erasure ec-profile
2020-01-29T19:46:41.845 INFO:tasks.workunit.client.0.smithi105.stderr:Invalid command: erasure doesn't represent an int
2020-01-29T19:46:41.845 INFO:tasks.workunit.client.0.smithi105.stderr:osd pool create <poolname> <int[0-]> {<int[0-]>} {replicated|erasure} {<erasure_code_profile>} {<rule>} {<int>} {<int>} {<int[0-]>} {<int[0-]>} {<float[0.0-1.0]>} :  create pool
2020-01-29T19:46:41.846 INFO:tasks.workunit.client.0.smithi105.stderr:Error EINVAL: invalid command
```

Reply:
oops, yeah--need to add a pg_num in there (say, 8)

Reply:
done